dockerfile

```diff
@@ -0,0 +1,27 @@
+
+FROM python:3.10
+
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+ENV PYTHONPATH /
+
+# Get necessary system packages
+RUN apt-get update \
+  && apt-get install --no-install-recommends --yes \
+     build-essential \
+     python3 \
+     python3-pip \
+     python3-dev \
+     mariadb-client \
+  && rm -rf /var/lib/apt/lists/*
+
+# Get necessary python libraries
+COPY requirements.txt .
+#COPY yoursql file
+#COPY your bash script
+RUN pip3 install --compile --no-cache-dir -r requirements.txt
```

Comment:
So this line hit an error for me. I ran docker-compose up and it made it this far on it's own.

`ERROR [5/5] RUN pip3 install --compile --no-cache-dir -r requirements.txt`

And it looks like this is the error:

`#0 3.571   × python setup.py egg_info did not run successfully.
#0 3.571   │ exit code: 1
#0 3.571   ╰─> [8 lines of output]
#0 3.571       Traceback (most recent call last):
#0 3.571         File "<string>", line 2, in <module>
#0 3.571         File "<pip-setuptools-caller>", line 34, in <module>
#0 3.571         File "/tmp/pip-install-xbnobixv/mariadb_8589f11f4b2c4f67bf12defd22cbedba/setup.py", line 27, in <module>
#0 3.571           cfg = get_config(options)
#0 3.571         File "/tmp/pip-install-xbnobixv/mariadb_8589f11f4b2c4f67bf12defd22cbedba/mariadb_posix.py", line 64, in get_config
#0 3.571           print('MariaDB Connector/Python requires MariaDB Connector/C '
#0 3.571       TypeError: not enough arguments for format string
#0 3.571       [end of output]`

The fix is easy here. Just get rid of the mariadb stuff. For this assignment we don't need to connect to the database with python anyway.